sway/desktop/output.c

```diff
@@ -509,7 +509,9 @@ static int output_repaint_timer_handler(void *data) {
 		fullscreen_con = workspace->current.fullscreen;
 	}
 
-	if (fullscreen_con && fullscreen_con->view && !debug.noscanout) {
+	if (fullscreen_con && fullscreen_con->view && !debug.noscanout
+			// Only output to monitor without compositing when saturation is changed
+			&& fullscreen_con->view->container->saturation == 1.0f) {
```

Comment:
Is it possible to simplify this to `fullscreen_con->saturation == 1.0f`?

Reply:
Oh... I guess that's what happens if you code for too long 😅